src/open_inwoner/configurations/admin.py

```diff
@@ -339,6 +354,76 @@ def _get_column(props):
             ),
         )
 
+    def javascript_preview(self, obj):
+        """Show preview of the uploaded script"""
+        if obj.custom_javascript:
```

Comment:
```suggestion
        if not obj.custom_javascript:
            return _("No JavaScript file uploaded")
```
Then you don't have to indend the whole file.

Reply:
"indend"?

Reply:
Indent!

Reply:
I meant: if you get the guard out of the way quickly, the rest of the function can be at regular-indent level.

Reply:
Since we are not offering any previews anywhere else in the Admin, I am removing this whole functionality.